cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/task/CopyNativeLibTask.java

```diff
@@ -120,6 +122,26 @@ private void copyImportedJarsForModules(BuildContext buildContext, List<BLangPac
         }
     }
 
+    private void copyPlatformLibsForModules(PackageID packageID, ExecutableJar executableJar, Path project) {
+        List<Library> libraries = manifest.getPlatform().libraries;
+        if (libraries == null) {
+            return;
+        }
+        List<Library> libs = libraries.stream().filter(lib -> lib.getModules() == null ||
+                Arrays.asList(lib.getModules()).contains(packageID.name.value)).collect(Collectors.toList());
+        for (Library lib : libs) {
+            String libFilePath = lib.getPath();
+            if (libFilePath != null) {
+                Path nativeFile = project.resolve(Paths.get(libFilePath));
+                if (lib.getScope() != null && lib.getScope().equals("test")) {
```

Comment:
if lib.getScope() == null means, default version? based on the logic scope null values are put in to moduleLibs.

Reply:
yes. We don't need that non-null check here.